code/modules/spells/spell_types/touch/_touch.dm

```diff
@@ -170,24 +170,59 @@
 	SHOULD_NOT_OVERRIDE(TRUE) // DEFINITELY don't put effects here, put them in cast_on_hand_hit
 
 	if(!can_hit_with_hand(target, caster))
-		return
+		return FALSE
```

Comment:
You don't `return FALSE` from signals, you would `return NONE`. Functionally identical, yes, but just not "correct"